src/main/java/mate/academy/spring/controller/UserControllerImpl.java

```diff
@@ -0,0 +1,52 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.UserResponseDto;
+import mate.academy.spring.service.UserService;
+import mate.academy.spring.service.mapper.UserMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+public class UserControllerImpl implements UserController {
+    private final UserService userService;
+    private final UserMapper userMapper;
+
+    public UserControllerImpl(UserService userService, UserMapper userMapper) {
+        this.userService = userService;
+        this.userMapper = userMapper;
+    }
+
+    @Override
+    @GetMapping("/inject")
+    public String injectUsers() {
+        User bob = new User("Bob", "Smith");
+        User alice = new User("Alice", "Valentine");
+        User john = new User("John", "Markus");
+
+        userService.add(bob);
+        userService.add(alice);
+        userService.add(john);
+        return "Users successfully added to DB";
+    }
+
+    @Override
+    @GetMapping("/{userId}")
+    public UserResponseDto get(@PathVariable Long userId) {
+        return userMapper.parse(userService.get(userId));
```

Comment:
```suggestion
        return userMapper.parseToDto(userService.get(userId));
```
we'll have response and request DTOs, so it's good to specify which way you map the object

Reply:
Done